src/components/GameView/ScrapDialog.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+	<v-dialog
+		v-model="show"
+		max-width="750"
+		scrollable
+	>
+		<template
+			#activator="{on, attrs}"
+		>
+			<span
+				v-bind="attrs"
+				v-on="on"
+			>
+				<slot name="activator" />
+			</span>
+		</template>
+		<v-card
+			id="scrap-dialog"
+		>
+			<v-card-title class="d-flex justify-space-between">
+				<h1>Scrap Pile</h1>
+				<v-btn
+					icon
+					data-cy="close-scrap-dialog-x"
+					@click="show=false"
+				>
+					<v-icon large>
+						mdi-close
+					</v-icon>
+				</v-btn>
+			</v-card-title>
+			<v-card-text>
+				<div class="d-flex flex-wrap justify-center align-center my-8">
+					<card
+						v-for="(card, index) in scrap"
+						:key="card.id"
+						class="mx-1 my-1"
+						:suit="card.suit"
+						:rank="card.rank"
+						:data-scrap-dialog-card="`${card.rank}-${card.suit}`"
+						@click="selectCard(index)"
+					/>
+				</div>
```

Comment:
Let's add a placeholder for when there are no cards in the scrap:
```
				<div class="d-flex flex-wrap justify-center align-center my-8">
					<!-- Empty Placeholder -->
					<template v-if="scrap.length === 0">
						<div class="d-flex flex-column">
							<p>There are no cards in the scrap pile.</p>
							<v-icon x-large>mdi-cancel</v-icon>
						</div>
					</template>
					<!-- Cards in the scrap -->
					<card
						v-for="(card, index) in scrap"
						:key="card.id"
						class="mx-1 my-1"
						:suit="card.suit"
						:rank="card.rank"
						:data-scrap-dialog-card="`${card.rank}-${card.suit}`"
						@click="selectCard(index)"
					/>
				</div>
```
![image](https://user-images.githubusercontent.com/6520704/147938286-f7e1b84a-13bb-4d40-87ed-a3dc6d8b70d9.png)